### DISCUSSION TITLE
Where can I check my sweep result?

### DISCUSSION THREAD

**Discussion by marcuswang6** [facebookresearch/hydra#2245] [Q&A]
Hi,
I run hydra sweep but cannot check the results, is there any log for sweep results?
Should I output the results in the code for convenience like ` print(acc) `, or just `return acc` is enough for later check?
Thanks!

**Comment by Jasha10** [comment]
Hi @Waterkin,

By default, the sweep results are not logged.
I'd recommend using `logging` module to output the result:
```python
...

import logging
log = logging.getLogger(__name__)

@hydra.main(...)
def my_app(cfg):
    ...
    result = 123

    log.info(result)
    return result
```
This way, you can check the result of the job in the generated log file.
```text
$ cat multirun/2022-05-31/10-49-51/0/my_app.log
[2022-05-31 10:49:52,308][__main__][INFO] - 123
```
